1526224210586	Sync.LogManager	DEBUG	Flushing file log
1526224210598	Sync.LogManager	DEBUG	Log cleanup threshold time: 1525360210598
1526224210704	Sync.LogManager	DEBUG	Done deleting files.
1526224211522	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1526224211524	Sync.Service	DEBUG	User-Agent: Firefox/59.0.2 (Linux x86_64) FxSync/1.61.0.20180406121424.desktop
1526224211524	Sync.Service	INFO	Starting sync at 2018-05-13 22:10:11 in browser session qzKXTZzD26AL
1526224211526	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1526224211534	Sync.Status	INFO	Resetting Status.
1526224211534	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1526224211534	FirefoxAccounts	DEBUG	already verified
1526224211534	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1526224211536	FirefoxAccounts	DEBUG	enter getAssertion()
1526224211542	FirefoxAccounts	DEBUG	got keyPair
1526224211542	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1526224211546	Services.Common.RESTRequest	TRACE	local time, offset: 1526224211546, -144
1526224211548	Services.Common.RESTRequest	TRACE	hawk auth header: Hawk id="8f2c15a9e6ad8ecc1d8d6f4a12755331d5216d944213fd98a1da8a48126119b8", ts="1526224211", nonce="JUTPrdxjYVg=", hash="XCGL7ksW9EPlqva3Zcdv59vMTaqFeeGiihKTAV2ufRA=", mac="e/2TmlX9AT2uKO0es37s7P0jWTL0FpQjTLzelgNT+Pc="
1526224211548	Services.Common.RESTRequest	DEBUG	POST request to https://api.accounts.firefox.com/v1/certificate/sign
1526224211548	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1526224211548	Services.Common.RESTRequest	TRACE	HTTP Header content-type: application/json
1526224211548	Services.Common.RESTRequest	TRACE	HTTP Header accept-language: en-US, en
1526224211548	Services.Common.RESTRequest	DEBUG	POST Length: 888
1526224211548	Services.Common.RESTRequest	TRACE	POST Body: {"publicKey":{"algorithm":"DS","y":"3D9E15B1B9BFE319F24159E3CEA91B37362266E7F3DB5AF9F1DEBCB0EAF6C108B3C6D17690967A1A2BB150005917466252E629BC258F1238583EC37647D7B5FB282D70168C3201903DD65CC430A048CDD82E8FB239D50DB3E1586366A1FF5B817DDC8A322B085CD8BE6DD878E1C73D74BD700D66633F664167C01845447C1448","p":"FF600483DB6ABFC5B45EAB78594B3533D550D9F1BF2A992A7A8DAA6DC34F8045AD4E6E0C429D334EEEAAEFD7E23D4810BE00E4CC1492CBA325BA81FF2D5A5B305A8D17EB3BF4A06A349D392E00D329744A5179380344E82A18C47933438F891E22AEEF812D69C8F75E326CB70EA000C3F776DFDBD604638C2EF717FC26D02E17","q":"E21E04F911D1ED7991008ECAAB3BF775984309C3","g":"C52A4A0FF3B7E61FDF1867CE84138369A6154F4AFA92966E3C827E25CFA6CF508B90E5DE419E1337E07A2E9E2A3CD5DEA704D175F8EBF6AF397D69E110B96AFB17C7A03259329E4829B0D03BBC7896B15B4ADE53E130858CC34D96269AA89041F409136C7242A38895C9D5BCCAD4F389AF1D7A4BD1398BD072DFFA896233397A"},"duration":21600000}
1526224211554	Services.Common.RESTRequest	TRACE	onStartRequest: POST https://api.accounts.firefox.com/v1/certificate/sign
1526224211554	Services.Common.RESTRequest	TRACE	Channel for POST https://api.accounts.firefox.com/v1/certificate/sign returned status code 2152398878
1526224211554	Services.Common.RESTRequest	DEBUG	POST https://api.accounts.firefox.com/v1/certificate/sign failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1526224211554	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 483"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:483 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211554	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 667"  data: no] Stack trace: get status()@resource://services-common/rest.js:667 < _onComplete()@resource://services-common/hawkclient.js:227 < onComplete()@resource://services-common/hawkclient.js:287 < onStopRequest()@resource://services-common/rest.js:485 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211556	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 682"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:682 < _onComplete()@resource://services-common/hawkclient.js:230 < onComplete()@resource://services-common/hawkclient.js:287 < onStopRequest()@resource://services-common/rest.js:485 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211556	Hawk	DEBUG	(Response) /certificate/sign: code: null - Status text: null
1526224211556	Services.Common.RESTResponse	TRACE	Processing response headers.
1526224211556	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 714"  data: no] Stack trace: get headers()@resource://services-common/rest.js:714 < _maybeNotifyBackoff()@resource://services-common/hawkclient.js:326 < _onComplete()@resource://services-common/hawkclient.js:237 < onComplete()@resource://services-common/hawkclient.js:287 < onStopRequest()@resource://services-common/rest.js:485 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211556	Services.Common.RESTResponse	TRACE	Processing response headers.
1526224211556	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 714"  data: no] Stack trace: get headers()@resource://services-common/rest.js:714 < _maybeNotifyBackoff()@resource://services-common/hawkclient.js:326 < _onComplete()@resource://services-common/hawkclient.js:238 < onComplete()@resource://services-common/hawkclient.js:287 < onStopRequest()@resource://services-common/rest.js:485 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211558	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 682"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:682 < _constructError()@resource://services-common/hawkclient.js:125 < _onComplete()@resource://services-common/hawkclient.js:243 < onComplete()@resource://services-common/hawkclient.js:287 < onStopRequest()@resource://services-common/rest.js:485 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211558	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 667"  data: no] Stack trace: get status()@resource://services-common/rest.js:667 < _constructError()@resource://services-common/hawkclient.js:126 < _onComplete()@resource://services-common/hawkclient.js:243 < onComplete()@resource://services-common/hawkclient.js:287 < onStopRequest()@resource://services-common/rest.js:485 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211558	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 667"  data: no] Stack trace: get status()@resource://services-common/rest.js:667 < _constructError()@resource://services-common/hawkclient.js:127 < _onComplete()@resource://services-common/hawkclient.js:243 < onComplete()@resource://services-common/hawkclient.js:287 < onStopRequest()@resource://services-common/rest.js:485 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211560	Services.Common.RESTResponse	TRACE	Processing response headers.
1526224211560	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 714"  data: no] Stack trace: get headers()@resource://services-common/rest.js:714 < _constructError()@resource://services-common/hawkclient.js:132 < _onComplete()@resource://services-common/hawkclient.js:243 < onComplete()@resource://services-common/hawkclient.js:287 < onStopRequest()@resource://services-common/rest.js:485 < waitForSyncCallback()@resource://services-common/async.js:97 < makeSpinningCallback/callback.wait()@resource://services-common/async.js:166 < _getAuthenticationHeader()@resource://services-sync/browserid_identity.js:791 < _buildHeaders()@resource://services-sync/resource.js:104 < _createRequest()@resource://services-sync/resource.js:139 < _doRequest()@resource://services-sync/resource.js:167 < get()@resource://services-sync/resource.js:261 < _fetchInfo()@resource://services-sync/service.js:482 < sync()@resource://services-sync/stages/enginesync.js:72 < onNotify()@resource://services-sync/service.js:1106 < WrappedNotify()@resource://services-sync/util.js:181 < WrappedLock()@resource://services-sync/util.js:137 < _lockedSync()@resource://services-sync/service.js:1099 < sync/<()@resource://services-sync/service.js:1091 < WrappedCatch()@resource://services-sync/util.js:107 < sync()@resource://services-sync/service.js:1080
1526224211560	FirefoxAccounts	ERROR	error POSTing /certificate/sign: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 483\"  data: no]","message":null,"code":null,"errno":null}
1526224211560	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 483\"  data: no]","message":null,"code":null,"errno":null}
1526224211560	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 483\"  data: no]","message":null,"code":null,"errno":null}
1526224211560	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1526224211560	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1526224211560	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1526224211562	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 483\"  data: no]","message":null,"code":null,"errno":null}
1526224211566	Sync.Resource	WARN	GET request to https://sync-600-us-west-2.sync.services.mozilla.com/1.5/92728811/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526224211566	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1526224211566	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1526224211566	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526224211566	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1526224211566	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 3173724 ms.
1526224211566	Sync.SyncScheduler	DEBUG	Next sync in 3173724 ms. (why=client-backoff-schedule)
1526224211570	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526224211570	Sync.ErrorHandler	DEBUG	Addons installed: 8
1526224211570	Sync.ErrorHandler	DEBUG	 - Firefox Multi-Account Containers, version 6.0.0, id @testpilot-containers
1526224211570	Sync.ErrorHandler	DEBUG	 - GNOME Shell integration, version 10.1, id chrome-gnome-shell@gnome.org
1526224211570	Sync.ErrorHandler	DEBUG	 - Web Extension for Shaarli, version 2.0.0, id shaarli@imirhil.fr
1526224211570	Sync.ErrorHandler	DEBUG	 - DuckDuckGo Privacy Essentials, version 2018.5.1, id jid1-ZAdIEUB7XOzOJw@jetpack
1526224211570	Sync.ErrorHandler	DEBUG	 - uBlock Origin, version 1.16.4, id uBlock0@raymondhill.net
1526224211570	Sync.ErrorHandler	DEBUG	 - Sync Tab Groups, version 0.6.7, id sync-tab-groups@eric.masseran
1526224211570	Sync.ErrorHandler	DEBUG	 - FoxyProxy Standard, version 6.2, id foxyproxy@eric.h.jung
1526224211570	Sync.ErrorHandler	DEBUG	 - Zotero Connector, version 5.0.39, id zotero@chnm.gmu.edu
